en/docs/design/create-api/create-api-revisions.md

```diff
@@ -1,6 +1,33 @@
-# Create API Revisions
-
-If you are an API Publisher, there can be a situation where you need to keep track of the different deployments of your API. For this purpose, a new concept named **Revisions** has been introduced. The revisions of your API can be **deployed** to specific Gateway Environment(s) as needed. These revisions cannot be edited and you can even **restore** your API to a specific revision if you want to discard the latest changes.
+# API Revisions
+
+In previous versions of WSO2 API Manager, API publishing and deployment in gateways were tightly coupled. 
```

Comment:
```suggestion
In previous versions of WSO2 API Manager, API publishing and deployment in Gateways were tightly coupled. 
```
Let's use "Gateway" and "Developer Portal" everywhere for consistency. In general let's use title case for all component names.

Reply:
Fixed with 69e70d09ae83fd2d50bcce077e7483e6a0e368e3